index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
The below review is for your README file. I mark it as optional because the previous reviewer did not mention them, maybe as an oversight but these issues are very important things that you should work on.

- [ ] [OPTIONAL] For the README template you have used, kindly do the following to further customize it to suit this project: 

**_All sections of this README template are mandatory except the FAQ which is optional._**

   - Remove the Microverse logo. If you have your logo, you can use it but I would advise you to remove it. 🙏
   - Add a meaningful project title and description that reflects what this project is about. **Hello Microver** does not mean anything we both know about.
   - For the Key Features section, kindly highlight what this project is all about. This will make it easy for those who go through this repository to know what was implemented by reading the README.md file. 👍
   - Since this project does not require any demo, you can leave out the Live Demo section.
   - For the Getting Started Section, and the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:
        - Installation of Git
        - Installation of NPM
        - A working browser
        - Internet connection
        - A code editor like Visual Studio Code
        - Installation of Live Server plugging in Visual Studio Code
        - Any other tool you can think about
   - For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
   - For the Install section, kindly add instructions for setting up linters used in this project
   - For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
   - For future features, add at least things you will do in the future to improve this project.
   - For the acknowledgment section, please add some praise to anyone who helps in this project.
   - For the contributing, leave it as it is in the template
   - For the MIT section, kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE
https://choosealicense.com/licenses/mit/

Please note: Most of the README section will begin with `>` some text like this 👇🏾 . They are comments that should guide and be removed after you edit the section. Do not push these comments, remove them all.

```
> Describe your project in 1 or 2 sentences.
> Describe the tech stack and include only the relevant sections that apply to your project.
> Describe between 1-3 key features of the application.
....
```

Kindly implement all the changes listed above. It will make your README file look professional and it will give you an idea of how to set README files for most of the projects you will execute as a Microverse student. 👍 �